app/javascript/config.js

```diff
@@ -0,0 +1,45 @@
+import { useDispatch } from "react-redux";
+
+// Actions
+const LOAD = "my-app/greetings/LOAD";
+// const CREATE = 'my-app/widgets/CREATE';
+// const UPDATE = 'my-app/widgets/UPDATE';
+// const REMOVE = 'my-app/widgets/REMOVE';
```

Comment:
- [ ] Kindly remove all commented-out and un used/ unwanted pieces of code from your codebase. It makes it look messy and cluttered.